### DISCUSSION TITLE
zh/docs/master/benchmark

### DISCUSSION THREAD

**Discussion by giscus** [gorse-io/docs#109] [Giscus]
# zh/docs/master/benchmark

性能对比 此页比较了 Gorse 不同数据库后端的性能表现。 RESTful APIs 在对 RESTful API 进行基准测试之前，应首先配置好开发环境。然后，将工作目录更改为server并运行基准测试脚本： cd server bash bench_test.sh --cache redis --data mysql

https://gorse.io/zh/docs/master/benchmark.html

**Comment by 3Taste** [comment]
性能对比-数据库选型-表头，中文文档中出现数据库和缓存数据库图标缺失
